src/main/java/mate/util/ConnectionUtil.java

```diff
@@ -6,10 +6,11 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL =
+            "jdbc:mysql://localhost:3306/taxi_service_db?serverTimezone=UTC";
```

Comment:
I'm not sure, but I think URL name should match the schema name from "init_db.sql" + timezone